poutyne/framework/callbacks/tracker.py

```diff
@@ -0,0 +1,176 @@
+from typing import Dict, List, Tuple
+
+import numpy as np
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+from .callbacks import Callback
+
+
+class Tracker(Callback):
+    def __init__(self, keep_bias: bool = False) -> None:
+        super().__init__()
+
+        self.keep_bias = keep_bias
+        self.writer = None
+
+        self.layer_names = []
+        self.number_layers = 0
+
+        self.tracker = None
+
+    def on_train_batch_end(self, batch: int, logs: Dict):
+        # pylint: disable=unused-argument
```

Comment:
Why is this necessary if it is not in other methods and other callbacks in Poutyne?

Reply:
Don't know ask pylint.

Reply:
Well that's strange.